geometry_msgs/Polygon polygon
DynamicTrajectoryPoint[] trajectory
